CodeFormatCore/src/Format/Analyzer/SemicolonAnalyzer.cpp

```diff
@@ -2,6 +2,7 @@
 #include "CodeFormatCore/Config/LuaStyleEnum.h"
 #include "CodeFormatCore/Format/FormatState.h"
 #include "LuaParser/Lexer/LuaTokenTypeDetail.h"
+#include <iostream>
```

Comment:
why include `<iostream>`

Reply:
sorry, that slipped through the cleanup!